types.cc

```diff
@@ -2368,8 +2368,13 @@ struct from_string_visitor {
             throw marshal_exception(format("Value not compatible with type {}: '{}'", ascii_type_name, s));
         }
     }
-    bytes operator()(const string_type_impl&) {
-        return to_bytes(bytes_view(reinterpret_cast<const int8_t*>(s.begin()), s.size()));
+    bytes operator()(const utf8_type_impl&) {
```

Comment:
Is the original implementation for string_type_impl no longer needed?

Reply:
probably string_type_impl extends utf8_type_impl so it will pick utf8_type_impl overload

Reply:
yes, ascii and utf8 combined exhaust the string type